fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/DruidClientConfigHelper.java

```diff
@@ -96,18 +128,46 @@ public static Integer getDruidNonUiPriority() {
      * Fetches the druid UI URL.
      *
      * @return druid UI URL
+     * @deprecated
```

Comment:
Add a newline after `@return` and then add a description saying to use `{@link #getDruidUrl()}`